[DOCS] Adds clarification of `discard_failed_expectations` to 0.18.x OSS quickstart (#9782)
great-expectations/great_expectations: docs/docusaurus/versioned_docs/version-0.18/oss/tutorials/quickstart.md

  The second <TechnicalTag tag="expectation" text="Expectation"/> uses explicit kwargs along with the `passenger_count` column.

>   The basic workflow when creating an Expectation Suite is to populate it with Expectations that accurately describe the state of the associated data.  Therefore, when an Expectation Suite is saved failed Expectations are not kept by default.  However, the `discard_failed_expectations` parameter of `save_expectation_suite(...)` can be used to override this behavior if you have created Expectations that describe the ideal state of your data rather than its current state.  

Reviewer: ```suggestion
When you add Expectations that describe the current data state to an Expectation Suite, failed Expectations are not kept by default when you save the Expectation Suite. If you have Expectations that describe an ideal data state, you can use the `discard_failed_expectations` parameter with `save_expectation_suite(...)` to override this behavior. 
```
